dev/ci/ci-sf.sh

```diff
@@ -13,6 +13,8 @@ tar xvfz vfa.tgz
 
 sed -i.bak '1i From Coq Require Extraction.' lf/Extraction.v
 sed -i.bak '1i From Coq Require Extraction.' vfa/Extract.v
+# Delete useless calls to try omega; unfold
+sed -i.bak '677,680d' vfa/SearchTree.v
```

Comment:
This is highly less robust than what I was proposing: if the exact position of these four lines changes in terms of line-number, then this will break and the resulting error won't even be easy to interpret.